### PR TITLE
A few minor changes

### PR DESCRIPTION
**Pull request by SawyerRuch**
Changes made to:
     - Sxhkdrc
     - Polybar config.ini
     - Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,5 +28,9 @@ https://forum.endeavouros.com/c/desktop-environments/bspwm/75
 - Text-Editor = xed
 - Application Launcher = Rofi
 
+## Get involved in our [forums](https://forum.endeavouros.com/c/desktop-environments/bspwm/75)
+
+<br>
```
Comment (flyingcakes85) [inline README.md:33]: We don't need html tag here. An empty line is sufficient.

File: bspwm.png
Comment (flyingcakes85) [inline bspwm.png:1]: This is something I will have to look into. Icon fonts seem to be broken.

**Review by flyingcakes85 — CHANGES_REQUESTED** [review]
Thanks for this PR. There's minor change I've requested in the README.

Currently not merging your PR, since the screenshot is doesn't show icon fonts properly. I'll have to look into why they're broken. Will merge your PR once we have a new screenshot with icons showing up.

File: README.md (hunk 1 above)
Comment (SawyerRuch) [reply README.md:33]: Ok. Will change. I will try to find a new image if you need.

File: bspwm.png
Comment (flyingcakes85) [reply bspwm.png:1]: image

@HabGesehen Please see this screenshot. This is a fresh install of EndeavourOS, where icons on Polybar are correctly rendered. I suspect your configuration has an error.

**Pull request closed without merging**